configs/cargo-deny/clarify.toml

```diff
@@ -0,0 +1,111 @@
+[clarify.askalono]
```

Comment:
What package or workspace is this clarify file for?

Reply:
cargo-deny itself :)

Reply:
🤯

Reply:
To help clarify, this is exactly where it's used :)

https://github.com/bottlerocket-os/bottlerocket-sdk/pull/49/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R483-R493